### PR TITLE
First round of security audit question and answers

### PR DESCRIPTION
**Pull request by cji**
Adding in the first round of question and answers received during the security audit RFP period.

cc @joelsmith @aasmall

### CODE REVIEW

File: wg-security-audit/RFP.md [hunk 1]
```diff
@@ -96,4 +96,11 @@ The audit should result in the following artifacts, which will be made public af
...
...ed reference architecture".<br> -	Is the expectation that this will be based on a specific k8s install mechanism (e.g. kubeadm)? <br> -	On a related note is it expected that High Availability configurations (e.g. multiple control plane nodes) should be included.<br> -	The assessment mentions Networking as a focus area.  Should a specific set of network plugins (e.g. weave, calico, flannel) be considered as in-scope or are all areas outside of the core Kubernetes code for this out of scope.<br> -	Where features of Kubernetes have been deprecated but not removed in 1.12, should they be considered in-scope or not? | 1. No, we are interested in the final topology -- the installation mechanism, as well as its default configuration, is tangental. The purpose is to contextualise the findings.<br>2. High-availability configurations should be included. For confinement of level of effort, vendor could create one single-master configuration and one high-availability configuration.<br>3. All plugins are out of scope per the bug bounty scope -- for clarification regarding the interface to plug-ins, please see the previous question.<br> 4. Deprecated features should be considered out of scope" |
```
Comment (aasmall) [inline wg-security-audit/RFP.md]: There seems to be a trailing double-quote at the end of point 2.

Comment (cji) [reply wg-security-audit/RFP.md]: Thanks!

File: wg-security-audit/RFP.md [hunk 2]
```diff
@@ -96,4 +96,11 @@ The audit should result in the following artifacts, which will be made public af
 
 ## Q & A
 
-This section intentionally left empty until the question period opens.
+| # | Question | Answer |
+|---|----------|--------|
+| 1 | The RFP says that any area included in the out of scope section of the k8s bug bounty programme is not in-scope of this review.  There are some areas which are out of scope of the bug bounty which would appear to be relatively core to k8s, for example Kubernetes on Windows. Can we have 100% confirmation that these areas are out of scope? | Yes. If you encounter a vulnerability in Kubernetes' use of an out-of-scope element, like etcd or the container network interface (to Calico, Weave, Flannel, ...), that is in scope. If you encounter a direct vulnerability in a third-party component during the audit, you can report it to them.  |
```
Comment (aasmall) [inline wg-security-audit/RFP.md]: Perhaps we should be the ones to handle comms with third parties.

Comment (cji) [reply wg-security-audit/RFP.md]: Yeah I think we say this better in question 3. Maybe copy that language into this?

'If you encounter a direct vulnerability in a third-party component during the audit you should follow the embargo section of the RFP.'

Comment (aasmall) [reply wg-security-audit/RFP.md]: perfect

**Pull request merged**